src/scss/_base.scss

```diff
@@ -1,80 +1,58 @@
-////
-/// @group o-table
-/// @link http://registry.origami.ft.com/components/o-table
-////
-
-/// Base tables styles - add to <table>
+/// Base tables styles and simple utility classes.
 @mixin oTableBase() {
-	@include oTypographyBody;
-	border-collapse: collapse;
-	border-spacing: 0;
-	margin: 0 0 20px;
-	width: 100%;
-
-	caption {
+	.o-table {
+		@include oTypographyBody;
+		border-collapse: collapse;
+		border-spacing: 0;
 		margin: 0;
-		padding: 0;
 		width: 100%;
-	}
-
-	th {
-		@include oTypographySansBold(1);
-		color: _oTableGet('table-header-color');
-		background-color: _oTableGet('header-background');
-	}
 
-	&[data-o-table--js] thead th:not([data-o-table-heading-disable-sort]) {
-		cursor: pointer;
-		user-select: none;
-		padding-right: 20px;
-		&:after {
-			@include oIconsGetIcon('arrows-up-down', $container-width: 20,  $container-height: 20, $iconset-version: 1);
-			content: '';
-			margin-right: -20px;
+		th,
+		td {
+			padding: 10px;
+			text-align: left;
 			vertical-align: top;
 		}
-		// Show descending icon with DSC sort applied.
-		&[aria-sort='descending'] {
-			&:after {
-				@include oIconsGetIcon('arrow-down', $container-width: 20,  $container-height: 20, $iconset-version: 1);
-				vertical-align: top;
+
+		th {
+			@include oTypographySansBold(1);
+			color: _oTableGet('table-header-color');
+			background-color: _oTableGet('header-background');
+			&:not([scope=row]) {
+				vertical-align: bottom;
 			}
 		}
-		// Show ascending icon with ASC sort applied or on hover with no sort.
-		&[aria-sort='none']:hover,
-		&:not([aria-sort]):hover,
-		&[aria-sort='ascending'] {
-			&:after {
-				@include oIconsGetIcon('arrow-up', $container-width: 20,  $container-height: 20, $iconset-version: 1);
-				vertical-align: top;
+
+		td {
+			@include oTypographySans(1);
+			color: _oTableGet('table-data-color');
+			scroll-snap-align: none center;
+			&:empty:before {
+				@include oIconsGetIcon('minus', $container-width: 15,  $container-height: 15, $iconset-version: 1);
+				content: '';
+				vertical-align: middle;
 			}
 		}
-	}
 
-	td {
-		@include oTypographySans(1);
-		color: _oTableGet('table-data-color');
-		&:empty:before {
-			@include oIconsGetIcon('minus', $container-width: 15,  $container-height: 15, $iconset-version: 1);
-			content: '';
-			vertical-align: middle;
+		.o-table__caption {
+			text-align: left;
+			box-sizing: border-box;
+			margin: 0;
+			padding: 0 10px;
+			width: 100%;
 		}
-	}
 
-	th,
-	td {
-		padding: 8px;
-		text-align: left;
-		vertical-align: top;
-	}
+		.o-table__cell--numeric {
+			text-align: right;
+		}
 
-	th:not([scope=row]) {
-		vertical-align: bottom;
-	}
-}
+		.o-table__cell--content-secondary {
+			@include oTypographySans(0);
```

Comment:
if a font was applied to the whole table, we could use [`oTypographySize(0)`](https://github.com/Financial-Times/o-typography/blob/079112ec83b024da2a22e714ee5dba0be871a427/src/scss/_mixins.scss#L71)

Reply:
Updated to `oTypographySize(0)` 👍